src/screens/rewards/components/RewardsProgramStatus.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Box, Stack, Text } from '@/design-system';
+import { useDimensions } from '@/hooks';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+type Props = {
+  status: 'error' | 'finished';
+};
+
+export const RewardsProgramStatus: React.FC<Props> = ({ status }) => {
+  const { height } = useDimensions();
+  const { top } = useSafeAreaInsets();
+
+  const emoji = status === 'error' ? '😵' : '💸';
+  const title = status === 'error' ? 'Something went wrong' : 'Program ended';
+  const text =
+    status === 'error'
+      ? 'Please check your internet connection and check back later.'
```

Comment:
needs i18n ser

Reply:
I will hold it off for some time, it is just a placeholder, I will add todo comments. I need some designs for that from Christian, this is just my invention, I think we will end up with something else

Reply:
I did it, I left it because this screen will quite likely change, but I guess it makes sense to make it nice in case it is left this way